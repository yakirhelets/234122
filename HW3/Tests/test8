getApartments sourceFour nas 2
