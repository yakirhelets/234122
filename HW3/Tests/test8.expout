11 2000 050-0000001
11 2000 050-0000011
