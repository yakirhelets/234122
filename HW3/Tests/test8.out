11 2000 050-0000001
12 3000 050-0000001
